src/vaults/src/handlers/bribe-processor-handler.ts

```diff
@@ -0,0 +1,15 @@
+import { TreeDistribution } from '../../generated/BribeProcessor/VotiumBribeProcessor';
+import { BadgerTreeDistribution } from '../../generated/schema';
+import { BVECVX } from '../constants';
+
+export function handleTreeDistribution(event: TreeDistribution): void {
+  let entity = new BadgerTreeDistribution(event.transaction.from.toHex());
+
+  entity.id = event.transaction.hash.toHex() + '-' + event.logIndex.toString();
+  entity.token = event.params.token.toHex();
```

Comment:
`loadToken(t.toHexString()).id` so in the case this was the first time the graph ever saw this token it gets saved